### DISCUSSION TITLE
Rainviewer overlay

### DISCUSSION THREAD

**Discussion by Sonwon1** [osmandapp/OsmAnd#18045] [Q&A] [answered]
I am trying to follow this video, [Online Maps for OsmAnd Android](https://www.youtube.com/watch?v=KBZ1DJa7RMg&list=WL&index=35).

Does anyone know why the Rainviewer overlay does not show up in the list of overlay choices?

**Comment by mikehgentry** [comment]
That layer has probably been added manually. As of the latest stable version, you have to go to Map source -> Add manually to add one. Obviously you'll need to know the right parameters to feed it too.

I think it's intended for the 'Add manually' option to appear directly in the Overlays / Underlays menu eventually (it's needlessly confusing to only have it available for the main layer).

**Comment by Sonwon1** [comment, marked as answer]
> That layer has probably been added manually. As of the latest stable version, you have to go to Map source -> Add manually to add one. Obviously you'll need to know the right parameters to feed it too.
> 
> I think it's intended for the 'Add manually' option to appear directly in the Overlays / Underlays menu eventually (it's needlessly confusing to only have it available for the main layer).

Thank you very much! I think I can figure it out now if it is still possible.

**Comment by Sonwon1** [comment]
Got it working, all of the information you need can be found,

https://osmand.net/docs/user/map/raster-maps/#underlay-layer

https://www.youtube.com/watch?v=KBZ1DJa7RMg&list=WL&index=30&t=16s

https://anygis.ru/Web/Html/Osmand_en

**Comment by Sonwon1** [comment]
For some reason unknown to me Rainviewer is not updating the radar maps.
